documents/write_acls.go

```diff
@@ -12,6 +15,7 @@ import (
 // if both old and new are set, then it is an edit operation
 type changedField struct {
 	property, old, new []byte
+	name               string
```

Comment:
@mikiquantum added the name here. We will return the names for invalid transitions